libnethack/include/artilist.h

```diff
@@ -225,7 +263,8 @@ static const struct artifact const_artilist[] = {
       CREATE_PORTAL, A_NEUTRAL, PM_WIZARD, NON_PM, 4000L),
 
     A("The Ring of Power", RIN_SLOW_DIGESTION,
-      (SPFX_RESTR | SPFX_INTEL | SPFX_INVIS), 0, 0, //It speaks, it is just handled elsewhere (allmain)
+      (SPFX_RESTR | SPFX_INTEL | SPFX_INVIS | SPFX_WRPOWER),
+      SPFX_RPOWER, 0, //It speaks, it is just handled elsewhere (allmain)
```

Comment:
The ring of power now conveys `SPFX_RPOWER` while being carried, and `SPFX_WRPOWER` while being worn.